packages/verification/src/models/v1/requests/verification-status/verification-status-request-data.ts

```diff
@@ -6,7 +6,8 @@ export interface VerificationStatusByIdentityRequestData {
   /** For type `number` use a [E.164](https://community.sinch.com/t5/Glossary/E-164/ta-p/7537)-compatible phone number. */
   'endpoint': string;
   /** The method of the verification. */
-  'method': 'sms' | 'callout' | 'flashcall';
+  // TODO v2.0 - Remove 'callout' option
```

Comment:
Even if not yet planned, a created ticket reference(within an "_V2 Epic_") as reminder ? (and ticket could reference this source file)

Reply:
I'll create a confluence page to prepare the things to do for the 2.0, it will be easier to manage / edit.